template/faas-flow/handler.go

```diff
@@ -506,10 +506,10 @@ func forwardAsync(fhandler *flowHandler, currentNodeId string, result []byte) ([
 
 	client := &http.Client{}
 	res, resErr := client.Do(httpreq)
-	resdata, _ := ioutil.ReadAll(res.Body)
 	if resErr != nil {
 		return resdata, resErr
```

Comment:
`resdata` is not defined here. 
Does these code compile ?

Reply:
I'm sorry for my mistake... I'will build with test and then commit

Reply:
please `squash` the commits